main.go

```diff
@@ -343,52 +331,19 @@ func websocketHandler(w http.ResponseWriter, req *http.Request) {
 			e.ServerTimestamp = time.Now().UTC().Format("2006-01-02T15:04:05.000-0700Z")
 			err = json.Unmarshal(eventData, &e)
 			if err != nil {
-				log.ErrorR(req, err, log.Data{"data": string(eventData)})
+				log.Event(req.Context(), "error unmarshalling websocket message", log.Error(err), log.Data{"data": string(eventData)})
 				continue
 			}
-			log.Debug("client log", log.Data{"data": e})
+			log.Event(req.Context(), "client log", log.Data{"data": e})
 
 			err = c.WriteJSON(florenceServerEvent{"ack", eventID})
 			if err != nil {
-				log.ErrorR(req, err, nil)
+				log.Event(req.Context(), "error writing websocket ack", log.Error(err))
 			}
 		default:
-			log.DebugR(req, "unknown event type", log.Data{"type": eventType, "data": string(eventData)})
-		}
-
-		// err = c.WriteMessage(mt, message)
-		// if err != nil {
-		// 	log.ErrorR(req, err, nil)
-		// 	break
-		// }
-	}
-}
-
-// FIXME this is to fix go-ns reverse proxy replacing host name so that
-// babbage has correct values when using {{location.hostname}} in handlebars
-func createBabbageProxy(proxyURL *url.URL, directorFunc func(*http.Request)) http.Handler {
```

Comment:
Is this meant to have been removed?

Reply:
yeah, it now uses reverseProxy from go-ns, and it's the frontend-router now instead of babbage

Reply:
👍 just checking.